### PR TITLE
Bug fix: typo in Set traffic distribution modal title

### PR DESCRIPTION
**Pull request by vikram-raj**
Bug - https://jira.coreos.com/browse/ODC-2251

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const TrafficSplittingModal: React.FC<Props> = ({
 }) => {
   return (
     <form className="modal-content" onSubmit={handleSubmit}>
-      <ModalTitle>Set traffic distrubution</ModalTitle>
+      <ModalTitle>Set traffic distribution</ModalTitle>
```
Comment (christianvogt) [inline frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.tsx]:
```suggestion
      <ModalTitle>Set Traffic Distribution</ModalTitle>
```

Comment (vikram-raj) [reply frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.tsx]: Done.

**Pull request merged**